README.md

```diff
@@ -166,6 +166,91 @@ optional `--wait --timeout 15m` parameters in a deployment pipeline to see if th
+### Prerequisites
+- Install *Docker Desktop* locally (Tested with 4.10.1) and enable kubernetes in settings.
+It is also recommended to increase the available resources of Docker Desktop to 4 CPU cores and 8 GB of RAM to ensure a smooth operation. 
+Alternatively you may install *kubectl* separately, however that may be a more error-prone approach.
+- Clone this repository locally on your computer.
+
+### Step 1
+Generate a `kube-config` file which will be used for the helm deployment. Make sure to generate it
+in the same folder as the git repo. Quickest way to do so is:
+
+```
+kubectl config view --raw > <name-of-your-file>
+```
+
+### Step 2
+Have a helm environment set up. We recommend using a docker container for that in order to facilitate the usage of Helm
+and not have to worry about dependencies. However it is also possible to have a native Helm installation for our purposes.
+You may set up the docker container using the following command:
+
+```
+docker run -ti --entrypoint= -v $(pwd):/apps alpine/helm:latest sh
+```
+
+When using the native Helm installation bear in mind that the Helm version needs to be 3.9.x
+
+### Step 3
+Next we want to deploy the helm chart, but before we can do that we will need to pull the dependencies of our chart.
+
+```
+helm dependency update #pulls all the other charts that our chart uses
+helm upgrade -i --kubeconfig <name-of-your-kubeconfig-flie> -f values_docker.yaml --create-namespace -n <your-namespace> magento .
+```
+
+### Step 3.5
+Check the progress of the deployment in the host machine, *not in your docker container*
+
+```
+kubectl get pods -n <your-namespace> -w
+```
+
+### Step 4
+Once the deployment has fully finished all that is missing is an ingress-nginx, which we deploy through Helm as well.
+In the previously created docker container run the following command:
+
+```
+helm upgrade --kubeconfig kube-config --install ingress-nginx ingress-nginx --repo https://kubernetes.github.io/ingress-nginx --namespace ingress-nginx --create-namespace
+```
+
+The command above will deploy a ingress-nginx pod that has access to the `/etc/hosts file`.
+Now all we need to do is add an entry for our domain, which if unaltered is *magento.local*
+
+```
+echo "127.0.0.1 magento.local" >> /etc/hosts
+```
+
+### Step 5
+Navigate to `http://magento.local` in your browser and play around with your local magento installation!
+
+#### Disclaimer
+This guide and the `values_docker.yaml` file are configured for the *magento.local* domain. If you wish to use a different domain,
+then you would need to update a few lines on the said file: 
+
+```
+magento:
+  env:
+    - name: MAGENTO_CLOUD_ROUTES
+      value: <base64-encoded-string-containing-your-domain>
+
+cronjob:
+  env:
+    - name: MAGENTO_CLOUD_ROUTES
+      value: <base64-encoded-string-containing-your-domain>
+
+ingress:
+  hosts:
+    - name: <your-domain>
+```
+
+Easiest way to generate the base64 encoded strings is to use the ones we provide as a base.
```

Comment:
@phoenix-henri well, this sentence redundant. Better reference the ece-tools section https://github.com/PHOENIX-MEDIA/magento2-helm#magento-ece-tools , which explains how to encode the string.